test/core/rules/std_test.py

```diff
@@ -35,6 +37,9 @@ def assert_rule_fail_in_sql(code, sql, configs=None):
     linted = Linter(config=cfg).lint_string(sql, fix=True)
     lerrs = linted.get_violations()
     print("Errors Found: {0}".format(lerrs))
+    parse_errors = list(filter(lambda v: type(v) == SQLParseError, lerrs))
+    if parse_errors:
+        pytest.fail(f"Found the following parse errors in test case: {parse_errors}")
```

Comment:
@NiallRees fyi, when I added the alternate case when syntax in the test I got an exception that SQLParseError doesn't have ".rule". So I added this conditional to fail the test in case something failed to parse

Reply:
Nice good idea 👍

Reply:
I think this is now redundant due to https://github.com/sqlfluff/sqlfluff/pull/648

Reply:
This isn't redundant I think, because #648 updates `assert_rule_pass_in_sql` to check that there are no violations, in my case here I'm updating `assert_rule_fail_in_sql` to make sure that none of the violations are parsing failures